src/ReactTouchHandler.js

```diff
@@ -78,7 +79,15 @@ class ReactTouchHandler {
         emptyFunction.thatReturnsFalse;
     }
 
+    // Can we just make this a boolean flag instead?
+    if (typeof preventDefault !== 'function') {
```

Comment:
yep, let's not support passing a function.  It doesn't take args, so I can't imagine what they would be using to decide...